project/Dependencies.scala

```diff
@@ -0,0 +1,43 @@
+import sbt.Keys.libraryDependencies
+import sbt._
+
+object Dependencies {
+
+  object ScalaVersions {
+    val scala211 = "2.11.12"
```

Comment:
I don't think this one is being used. Better remove to avoid confusion.

Reply:
Done